huawei_solar_battery_optimization.yaml

```diff
@@ -442,7 +444,7 @@ script:
           periods: |-
             00:01-06:00/1234567/+
             17:00-23:59/1234567/-
-          device_id: 8ee3ed713a270c3bbf9596bcfe414175
+          device_id: "{{ states('input_text.hsbo_device_id') }}"
```

Comment:
this one should be

device_id: "{{ states('input_text.hsbo_device_id_batteries') }}"